package.json

```diff
@@ -6,7 +6,7 @@
     "Ryan Cannon <ryan@ryancannon.com> (https://ryancannon.com)"
   ],
   "description": "Content Security Policy middleware.",
-  "version": "2.9.4",
+  "version": "2.9.5",
```

Comment:
Could you revert this change? I'll do the version bump in a separate commit.

Reply:
I have reverted this commit and pushed